manifests/broker.pp

```diff
@@ -0,0 +1,20 @@
+# Set up the broker
+class pulp::broker {
+  if $pulp::manage_broker {
```

Comment:
It's not very clear for me what you are asking, but if you want to say to remove the manage_db and manage_broker parameters, let me know. 

From my point of view, pulp shouldn't manage the mongodb or broker installations (or any configuration aspect for that matter of those two).